.github/workflows/github-ci.yml

```diff
@@ -178,6 +166,19 @@ jobs:
       # https://github.com/ukaea/parallel-preprocessor/releases/download/dev/parallel-preprocessor.zip
   
     ########################################## only for Ubuntu debug release ############################################
+    - name: Re-compile on Linux (debug + coverage)
+      working-directory: ${{ github.workspace }}
+      # env:
+      #         CC: ${{ matrix.compiler }}
+      #         CXX: ${{ matrix.compilerpp }}
+      run: |
+        rm -rf build
+        mkdir build

```

Comment:
I would use a different name for this build directory.  Although it shouldn't make any difference, I find it useful for avoiding errors which can come about when you have two different build directories in the same place